stack/loader.go

```diff
@@ -60,27 +68,20 @@ func (l Loader) Load(dir string) (S, error) {
 		return S{}, fmt.Errorf("stack %q is not a leaf directory", dir)
 	}
 
-	l.set(dir, cfg.Stack)
-	return l[dir], nil
-}
-
-// LoadChanged is like Load but sets the stack as changed if loaded
-// successfully.
-func (l Loader) LoadChanged(dir string) (S, error) {
-	s, err := l.Load(dir)
-	if err != nil {
-		return S{}, err
-	}
-
-	s.changed = true
-	return s, nil
+	l.set(stackpath, cfg.Stack)
+	return l.stacks[stackpath], nil
 }
 
 // TryLoad tries to load a stack from directory. It returns found as true
 // only in the case that path contains a stack and it was correctly parsed.
 // It caches the stack for later use.
 func (l Loader) TryLoad(dir string) (stack S, found bool, err error) {
```

Comment:
Hmm on this Try* method we went with the more classical (return value, boolean, error), instead of (boolean, return value, error). I prefer it :-) (not that it matter much, again feel free to ignore)